master/usage/policy/secure-metrics.md

```diff
@@ -0,0 +1,408 @@
+---
+title: Securing access to Calico's metrics endpoints
+---
+
+## About securing access to {{site.prodname}}'s metrics endpoints
+
+When using {{ site.prodname }} with prometheus metrics enabled, it may be desirable to use network policy
+to limit access to {{ site.prodname }}'s metrics endpoints.
+
+### Purpose of this page
+
+Provide example workflows for creating network policies to limit access to {{site.prodname}}'s metrics endpoints.
+
+## Prerequisites
+
+- {{site.prodname}} is installed with [prometheus metrics reporting enabled](TODO).
+- calicoctl is [installed in your PATH and configured to access the data store](../calicoctl/install).
+
+## Using a blacklist approach
+
+This approach allows all traffic to your hosts by default, but lets you limit access to specific ports using
```

Comment:
I think it'd be helpful to include a sentence to help the user weigh up the two options.  "This approach is more secure because only explicitly-allowed traffic will get through, but it requires you to know all the ports that should be open on the host."